### PR TITLE
feat(3dtiles-implicit-tiling): parse implicit tiling structure

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
Parsing implicit tiles as common loaders.gl Tile3D structure **- Done**
Update tests with more checks **- In next PR**
Parse child tiles region bounding volume - **In next PR**

### CODE REVIEW

File: modules/3d-tiles/src/lib/utils/morton.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+// Morton Index specification - https://en.wikipedia.org/wiki/Z-order_curve
+// Code is taken from - https://github.com/thi-ng/umbrella/tree/develop/packages/morton
```
Comment (ibgreen) [inline modules/3d-tiles/src/lib/utils/morton.ts:2]: Specify under which OSS license this file was "forked" (Apache 2.0 in this case, which is fine).

Comment (ibgreen) [reply modules/3d-tiles/src/lib/utils/morton.ts:2]: Adding a util is fine, but please make an effort to also add a test: https://github.com/thi-ng/umbrella/tree/develop/packages/morton/test

**Review by ibgreen — APPROVED** [review]

File: modules/3d-tiles/src/lib/utils/morton.ts (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/3d-tiles/src/lib/utils/morton.ts]: Removed this part

File: modules/3d-tiles/src/types.ts [hunk 2]
```diff
@@ -71,6 +71,8 @@ type Availability = {
 
 export type ExplicitBitstream = Uint8Array;
 
+export type SubdivisionScheme = 'QUADTREE' | 'OCTREE';
```
[the remark this answers is not in the document]
Comment (dryabinin-actionengine) [reply modules/3d-tiles/src/types.ts:77]: Done

**Pull request merged**